rust/agents/relayer/src/msg/serial_submitter.rs

```diff
@@ -179,7 +180,9 @@ async fn prepare_task(
                 debug!(?op, "Operation prepared");
                 metrics.ops_prepared.inc();
                 // this send will pause this task if the submitter is not ready to accept yet
-                tx_submit.send(op).await?;
+                if let Err(err) = tx_submit.send(op).await {
```

Comment:
this error can only occur if the receiving end has been closed. I'm only logging the error rather than propagating / panicking to avoid taking down the tasks for other chains